exercises/tom_clayton/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,158 @@
+/**
+ * Import statements
+ */
+
+#include <iostream>
+#include <sstream>
+#include <fstream>
+#include <algorithm>
+#include <cctype>
+#include <string.h>
+#include <vector>
+#include <map>
+using namespace std;
+
+/**
+ * Function to return a string filepath from user input
+ */
+
+std::string getInputFn() {
+  std::cout << "Please enter the path of the file to be analysed" << std::endl;
+  std::string fin;
+  std::cin >> fin;
+  return fin;
+}
+
+/**
+ * Function to split up a string and return a vector of words with
+ * len <4, and converted to lower case
+ */
+
+std::vector<std::string> splitByDelim(const std::string& s, char delim) {
```

Comment:
Please can you add const to the 2nd parameter

Reply:
Done